README.rst

```diff
@@ -19,9 +19,8 @@ community:
 Plugin                               Status (*Experimental*, *Production*, or *Deprecated*)
 ===================================  ======================================================
 tutor-contrib-learner-dashboard-mfe  Experimental
-===================================  ======================================================
 tutor-contrib-blockstore-filesystem  Experimental
-===================================  ======================================================
+tutor-contrib-blockstore-minio       Experimental
```

Comment:
...but the tailing separator is still needed. :)

Reply:
(Yeah, RST, I know.)

Reply:
I hadn't installed the rst preview vscode extension on this machine since switching all my work over. Now I've got it again